libcnb-test/src/test_runner.rs

```diff
@@ -160,11 +159,42 @@ impl TestRunner {
         let test_context = TestContext {
             pack_stdout: output.stdout,
             pack_stderr: output.stderr,
-            image_name,
+            docker_resources,
             config: config.clone(),
             runner: self,
         };
 
         f(test_context);
     }
 }
+
+pub(crate) struct TemporaryDockerResources {
```

Comment:
I'm not super happy with the name `TemporaryDockerResources` (and also the `docker_resources` field name for `TestContext`), so open to ideas :-)

Another option might be `TemporaryBuildResources` - though "Build" is slightly overloaded in that `libcnb-test` has the `build` module and types like `BuildConfig` - which are actually referring to the Cargo build (ie compile) not the `pack build`.

(Though given this is an internal-only type I suppose it's not the end of the world if we can't find a perfect name.)

Reply:
FWIW: I think this is fine as-as. :)

Reply:
yeah, i see no problem with the current name.  it communicates to me that the image, build cache volume, and launch cache volume are meant to be temporary so i shouldn't see them still kicking around after a test executes.